README.md

```diff
@@ -23,11 +23,12 @@ setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 ```
- * Top branch needed to run updated double b tagger training on AK8 jets
+ * Top branch needed to run updated double b tagger training on AK8 jets, and latest Run II MET filters
 ```
 git fetch --tags btv-cmssw
 
 git cms-merge-topic -u cms-btv-pog:BoostedDoubleSVTaggerV3-WithWeightFiles-v1_from-CMSSW_8_0_8_patch1
```

Comment:
@jkarancs : thanks for the MET filter implementation. PR#55 is still open, do we merge it, or wait for these filters to be implemented in a later CMSSW release? It says that the bad muon and badCharged hadron filters are RECO problems, so these should disappear in the re-RECO, right?